modules/apps/commerce/commerce-cart-taglib/build.gradle

```diff
@@ -1,7 +1,7 @@
 dependencies {
-	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-annotations", version: "2.10.3"
-	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-core", version: "2.10.3"
-	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-databind", version: "2.10.5.1"
+	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-annotations", version: "2.12.4"
```

Comment:
Hi @yuhai ,
Let's try to get ride of the compileInclude usage like this, only use compileInclude when you are the only one need this jar.
jackson related jars are used in many places, let's move it to lib/portal and export the packages like this https://github.com/shuyangzhou/liferay-portal/pull/10218/commits/abfa60395a5000d754172bbced581bb23b499a2b